lib/api/v1/incident-controller.js

```diff
@@ -109,6 +139,8 @@ module.exports = function(app, user) {
         // Delete each incident explicitly to catch it in model
         incident.remove(function(err) {
           if (err) return res.send(err.status, err.message);
+          var logDetails = writelog.writeIncident(req, incident, 'deleteIncident');
```

Comment:
I don't want to modify the structure of logger too much. I have tried to avoid logger from accessing the database (therefore creating userLogger.js), or in this case, from accessing secrets.json.
What I did was call logger from writeLog, so there's no unnecessary returning.